tests/Microsoft.Identity.Test.Unit/PublicApiTests/TelemetryClientTests.cs

```diff
@@ -13,6 +18,24 @@ namespace Microsoft.Identity.Test.Unit.PublicApiTests
     [TestClass]
     public class TelemetryClientTests : TestBase
     {
+        private MockHttpAndServiceBundle _harness;
+        private PublicClientApplication _pca;
+        private ITelemetryClient _telemetryClient;
+
+        [TestInitialize]
+        public override void TestInitialize()
+        {
+            _telemetryClient = new TelemetryClient(TestConstants.ClientId);
+            base.TestInitialize();
+        }
+
+        [TestCleanup] 
+        public override void TestCleanup()
+        {
+            _harness?.Dispose();
```

Comment:
The harness should not be shared between tests, becasue the each test should consume all HTTP message from it.